spec/controllers/products_controller_spec.rb

```diff
@@ -2,14 +2,17 @@
 
 RSpec.describe ProductsController, type: :controller do
   let(:user) { create(:regular_user) }
+
   before(:all) do
     @product_one = create(:product, name: "testproduct1")
     @product_two = create(:product, name: "testproduct2")
   end
+
   after(:all) { DatabaseCleaner.clean_with(:truncation) }
 
   describe "making a review" do
     it "renders show template" do
+      user.update(verified: true)
```

Comment:
Could not be added because the user was not created in the login method, hence it could not reference it.